layouts/community/single.html

```diff
@@ -1,3 +1,20 @@
+{{/*
+  Copyright © 2015-2019 Cask Data, Inc.
```

Comment:
@dbuzhorkaa sorry I missed this earlier. Since these are all newly added files, can you change this copyright to:

```
Copyright © 2019 CDAP
```

Reply:
done